.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: AI Code Reviewer
+
+on: [pull_request]
+
+    # types:
+    #   - opened
+    #   - synchronize
+# permissions: write-all
```

Comment:
The `permissions: write-all` line is commented out. If it's not needed, please remove it.